src/components/customers/CustomerItem.tsx

```diff
@@ -152,7 +152,7 @@ const Item = styled(ListItemLink)`
 `
 
 const MediumCell = styled(Typography)`
-  width: 200px;
```

Comment:
I wonder if we couldn't let the "subscription" column shrink to have more space in this one ?

Reply:
Instead of the customer name one ? yep sounds better so the customer name had a min width instead.

Gonna update

Reply:
Done ! I've removed the min-width of the middle column, so it only takes the place it needs to take and customer name can be fully displayed